doc/changelog_update.md

```diff
@@ -0,0 +1,65 @@
+# Changelog Update
+
+If you want to help with updating the [changelog][changelog], you're in the right place.
+
+## When to update
+
+Typos and other small fixes/additions are _always_ welcome.
+
+Special care needs to be taken when it comes to updating the changelog for a new
+Rust release. For that purpose, the changelog is ideally updated during the week
+before an upcoming stable release. You can find the release dates on the [Rust
+Forge][forge].
+
+Most of the time we only need to update the changelog for minor Rust releases. It's
+been very rare that Clippy changes were included in a patch release.
+
+## How to update
+
+### 1. Finding the relevant Clippy commits
+
+Each Rust release ships with its own version of Clippy. The Clippy submodule can
+be found in the [tools][tools] directory of the Rust repository.
+
+To find the Clippy commit hash for a specific Rust release you select the Rust
+release tag from the dropdown and then check the commit of the Clippy directory:
+
```

Comment:
Since I was confused just now.. This should also explain how to find the Clippy commit up the upcoming stable release.